airbyte-integrations/connectors/destination-gcs/src/main/java/io/airbyte/integrations/destination/gcs/GcsStorageOperations.java

```diff
@@ -51,3 +51,4 @@ protected Map<String, String> getMetadataMapping() {
   }
 
 }
+
```

Comment:
this change seems random

Reply:
Made this to show the result. I'll remove it when changes will be approved

Reply:
✅ Removed